app/models/ems_refresh/save_inventory.rb

```diff
@@ -116,7 +116,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        found.send(:clear_association_cache)
+        found.reload
```

Comment:
> Or more specifically, reload will go refetch the attributes, so regardless we would have an object with the attributes inside

Yes, but the source of the attributes is the persisted values from the database as opposed to strings coming from the libraries.  The latter strings not being GC'd could leave other trees of library objects "live" and not GC'able.  It's a guess.